src/feat/pitch-functions.cc

```diff
@@ -856,14 +857,21 @@ void OnlinePitchFeatureImpl::ExtractFrame(
   } else {
     // frame is partly in the remainder and partly in the new part.
     int32 remainder_offset = downsampled_signal_remainder_.Dim() + offset;
-    KALDI_ASSERT(remainder_offset >= 0);  // or we didn't keep enough remainder.
+    /*KALDI_ASSERT(remainder_offset >= 0);*/  // or we didn't keep enough remainder.
```

Comment:
This bit has now been restored to previous version as border case is now treated separately